.github/workflows/winget.yml

```
@@ -0,0 +1,84 @@
+name: Publish to WinGet
+
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-winget:
+    name: Publish to WinGet
+    runs-on: windows-latest
+    # Only run for CLI releases (not JS API releases)
+    if: startsWith(github.event.release.tag_name, '@biomejs/biome@')
+    steps:
+      - name: Extract version from tag
+        id: extract-version
+        run: |
+          $tag = "${{ github.event.release.tag_name }}"
+          $version = $tag -replace '^@biomejs/biome@', ''
+          echo "version=$version" >> $env:GITHUB_OUTPUT
+          echo "Extracted version: $version"
+        shell: pwsh
+
+      - name: Install .NET Runtime
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '6.x'
```

`.NET 6` LTS support ends on [November 12, 2024](https://devblogs.microsoft.com/dotnet/dotnet-6-end-of-support/) Should we bump the workflow to `.NET 8` (current LTS) instead?

```suggestion
          dotnet-version: '8.x'
```